canvas_modules/common-canvas/__tests__/test_resources/paramDefs/toggle_paramDef.json

```diff
@@ -0,0 +1,429 @@
+{
+  "titleDefinition": {
+    "title": "toggle"
+  },
+  "current_parameters": {
+    "toggle": true,
+    "toggle_null": null,
+    "toggle_table": [
+      [
+        true
+      ]
+    ],
+    "toggle_table_cond": [
+      []
+    ],
+    "toggle_error": false,
+    "toggle_warning": false,
+    "disable": true,
+    "hide": true,
+    "toggle_hidden": false
+  },
+  "parameters": [
+    {
+      "id": "toggle",
+      "type": "boolean"
+    },
+    {
+      "id": "toggle_null",
+      "type": "boolean"
+    },
+    {
+      "id": "toggle_table",
+      "type": "array[togglees]"
+    },
+    {
+      "id": "toggle_table_cond",
+      "type": "array[togglees]"
+    },
+    {
+      "id": "toggle_default",
+      "default": true,
+      "type": "boolean"
+    },
+    {
+      "id": "toggle_undefined",
+      "type": "boolean"
+    },
+    {
+      "id": "toggle_error",
+      "type": "boolean"
+    },
+    {
+      "id": "toggle_warning",
+      "type": "boolean"
+    },
+    {
+      "id": "disable",
+      "type": "boolean"
+    },
+    {
+      "id": "toggle_disable",
+      "type": "boolean",
+      "required": true
+    },
+    {
+      "id": "hide",
+      "type": "boolean"
+    },
+    {
+      "id": "toggle_hidden",
+      "type": "boolean",
+      "required": true
+    }
+  ],
+  "complex_types": [
+    {
+      "id": "togglees",
```

Comment:
spelling

Reply:
<img width="377" alt="image" src="https://user-images.githubusercontent.com/89609176/187748325-a8854459-b157-4b7a-8840-2df468d144b4.png">

Reply:
This is wrong in the PR yet.  Maybe you fix it in the harness and didn't sync it in the test case.